services/spar/src/Spar/Scim/Types.hs

```diff
@@ -96,4 +87,4 @@ normalizeLikeStored usr =
     tweakExtra = ScimUserExtra . RichInfo . normalizeRichInfoAssocList . unRichInfo . view sueRichInfo
 
     tweakActive :: Maybe Scim.ScimBool -> Maybe Scim.ScimBool
-    tweakActive = fmap Scim.ScimBool . maybe (Just True) Just . fmap Scim.unScimBool
+    tweakActive = fmap Scim.ScimBool . maybe (Just True) (Just . Scim.unScimBool)
```

Comment:
This doesn't look like an improvement, it's much harder to understand this code now. How about changing it to `Just . Scim.ScimBool . fromMaybe True . fmap Scim.unScimBool`?

Reply:
Isn't that the same as…
```haskell
Just . Scim.ScimBool . maybe True Scim.unScimBool
```

Reply:
Yes, that's better!